config/samples/_v1alpha1_orchestrator.yaml

```diff
@@ -6,76 +6,29 @@ metadata:
-      name: rhdh # name of the operator package
-      sourceName: redhat-operators # name of the catalog source
-      startingCSV: "" # The initial version of the operator
-      targetNamespace: rhdh-operator # the target namespace for the backstage CR in which RHDH instance is created
-  rhdhPlugins: # RHDH plugins required for the Orchestrator
-    npmRegistry: "https://npm.registry.redhat.com" # NPM registry is defined already in the container, but sometimes the registry need to be modified to use different versions of the plugin, for example: staging(https://npm.stage.registry.redhat.com) or development repositories
-    scope: "@redhat"
-    notificationsConfig:
-      enabled: false # whether to install the notifications email plugin. requires setting of hostname and credentials in backstage secret to enable. See value backstage-backend-auth-secret. See plugin configuration at https://github.com/backstage/backstage/blob/master/plugins/notifications-backend-module-email/config.d.ts
-      port: 587 # SMTP server port
-      sender: "" # the email sender address
-      replyTo: "" # reply-to address
-
+        enabled: false # Determines whether to install the Notifications Email plugin. Requires setting of hostname and credentials in backstage secret. The secret, backstage-backend-auth-secret, is created as a pre-requisite. See value backstage-backend-auth-secret. See plugin configuration at https://github.com/backstage/backstage/blob/master/plugins/notifications-backend-module-email/config.d.ts
+        port: 587 # SMTP server port. Defaults to 587. Optional
+        sender: "" # Email address of the Sender. Defaults to empty string. Optional
+        replyTo: "" # Email address of the Recipient. Defaults to empty string. Optional
   postgres:
     serviceName: "sonataflow-psql-postgresql" # The name of the Postgres DB service to be used by platform services. Cannot be empty.
     serviceNamespace: "sonataflow-infra" # The namespace of the Postgres DB service to be used by platform services.
     authSecret:
-      name: "sonataflow-psql-postgresql" # name of existing secret to use for PostgreSQL credentials.
-      userKey: postgres-username # name of key in existing secret to use for PostgreSQL credentials.
-      passwordKey: postgres-password # name of key in existing secret to use for PostgreSQL credentials.
-    database: sonataflow # existing database instance used by data index and job service
+      name: "sonataflow-psql-postgresql" # Name of existing secret to use for PostgreSQL credentials. Required
+      userKey: postgres-username # Name of key in existing secret to use for PostgreSQL credentials. Required
+      passwordKey: postgres-password # Name of key in existing secret to use for PostgreSQL credentials. Required
+    database: sonataflow # Name of key in existing secret to use for PostgreSQL credentials. Required
   orchestrator:
     namespace: "sonataflow-infra"
```

Comment:
as commented in the ADR, this confuses me: what does `orchestrator` refer to here? The sonataflow manifests?

Reply:
Yes, the `orchestrator` is parent to the `sonataflowplatform` configuration and `namespace`. It confused me to that's why there's suggestion to move those under the `serverlessLogicOperator` field and split that field. Still waiting for feedback that's why I haven't made those changes yet.

Reply:
this will be mimicking the sontaflowplatform CRD, which is great IMO

Reply:
As agreed on the ADR side, updated field to this.
```
  "platform": {
    "namespace": "sonataflow-infra",
    "resources": {
      "limits": {
        "cpu": "64Mi",
        "memory": "250m"
      },
      "requests": {
        "cpu": "1Gi",
        "memory": "500m"
      }
    },
    "eventing": {
      "broker": {
        "name": "my-knative",
        "namespace": "knative"
      }
    }
  }
```